.github/workflows/ci.yml

```diff
@@ -0,0 +1,211 @@
+---
+name: "CI"
+on:
+  - "push"
+  - "pull_request"
+
+jobs:
+  black:
+    runs-on: "ubuntu-20.04"
+    env:
+      INVOKE_NAUTOBOT_CHATOPS_LOCAL: "True"
+    steps:
+      - name: "Check out repository code"
+        uses: "actions/checkout@v2"
+      - name: "Set up Python"
+        uses: "actions/setup-python@v2"
+        with:
+          python-version: "3.9"
+      - name: "Install Python Packages for Setup"
+        run: "pip install invoke black"
```

Comment:
This should be from the poetry installed version. The poetry lock file maintains the version of Black for the project. Black itself updates somewhat regularly and then will cause lots of confusion.

Reply:
ok, then maybe we could just install the Poetry dev dependencies with `--dev-only`, so the installation process wil lbe lighter.

Reply:
Do you mean this just for `black` or other tools as well (`yamllint`, `bandit`, `pydocstyle`)? The reason I am asking this is that using `pip` saves a lot of time... 

On the other hand, do we care if we use the version `21.8b0` or `21.4b0` for example?

I also checked the cookiecutter and we have `black = "*"` in the `pyproject.toml`. So I guess we are using a different versions of `black` for different projects already (based on the time when the `poetry.lock` is created).

Reply:
> --dev-on

@chadell  I didn't see your message before. But `--dev-only` will be available in poetry `1.2`.

Reply:
> ok, then maybe we could just install the Poetry dev dependencies with `--dev-only`, so the installation process wil lbe lighter.

@chadell I didn't see your message before. But `--dev-only` will be available in poetry `1.2`.